build/ci/golangci.yml

```diff
@@ -121,3 +122,7 @@ issues:
         - dupl
         - dogsled
         - bodyclose
+    # False positive: https://github.com/kunwardeep/paralleltest/issues/8.
+    - linters:
+        - paralleltest
+      text: "Range statement for test TestFunctional does not use range value in test Run"
```

Comment:
Funkcionalne testy tiez bezia paralelne a je to tam spravne, ale linter to nejak nevedel spravne spracovat.